ci/test-pipeline.yml

```diff
@@ -84,7 +84,7 @@ stages:
             inputs:
               mavenPomFile: "core/pom.xml"
               mavenOptions: "-Xmx3072m"
-              options: "compile -DskipTests"
+              options: "compile -DskipTests -P build-web-console,use-built-in-nodejs"
```

Comment:
This will run when there is no java changes only, you prob want this on other maven steps too